src/Driver/PgSQL/Connection.php

```diff
@@ -101,28 +103,19 @@ public function lastInsertId($name = null)
         return $this->query('SELECT LASTVAL()')->fetchOne();
     }
 
-    /** @return true */
```

Comment:
Is it actually possible to get `false` now or would the driver throw in case of an error?